src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php

```diff
@@ -54,6 +54,9 @@ protected function instantiateController($class)
         try {
             return parent::instantiateController($class);
         } catch (\Error $e) {
+            if (0 === strpos($class, '\\')) {
```

Comment:
Instead of deprecating this possibility, what about trimming the starting backslash in the `instantiateController` method?